NAMESPACE

```diff
@@ -11,7 +11,6 @@ export(dashbioOncoPrint)
 export(dashbioSequenceViewer)
 export(dashbioSpeck)
 export(dashbioManhattan)
+export(dashbioAlignmentViewer)
```

Comment:
Hm, I don't think this should be here -- IIRC we got rid of the AlignmentViewer component in #425

Reply:
Hmm, strange indeed. The generator should blow away the existing `R` and `man` directories before rebuilding the `NAMESPACE` file IIRC, not sure why that didn't happen. I'll take care of this now, thanks for catching it.

Reply:
fixed in 6e4f9c5